internal/compiler/widgets/fluent-base/std-widgets-impl.slint

```diff
@@ -82,14 +82,15 @@ export global StyleMetrics := {
 
 export Button := Rectangle {
     callback clicked;
-    property<string> text <=> text.text;
-    property<bool> has-focus: fs.has-focus;
-    property<bool> pressed: self.enabled && touch.pressed;
-    property<bool> enabled <=> touch.enabled;
-    property<bool> checkable;
-    property<bool> checked;
-    property<image> icon;
-    property<length> font-size <=> text.font-size;
+    in property<string> text <=> text.text;
+    out property<bool> has-focus: fs.has-focus;
+    out property<bool> pressed: self.enabled && touch.pressed;
+    in property<bool> enabled <=> touch.enabled;
+    in property<bool> checkable;
+    in-out property<bool> checked;
+    in property<image> icon;
+    // FIXME: remove
```

Comment:
I'm curious, why this FIXME? Should the comment perhaps explain why it should be removed? Is this because for `TextEdit` and `LineEdit` we should have them but not for `Button`, `SpinBox` or anything else where text is just a sub-element?

(I'd agree with limiting this to `TextEdit`/`LineEdit` - not sure this needs to be configurable in the others at this point)

Maybe worth a breaking-change labelled issue for 1.0.0?

Reply:
This property is not documented, and is also not working in the native style.